docs/Doxyfile

```diff
@@ -38,7 +38,7 @@ PROJECT_NAME           = "SMART on FHIR Python Client"
 # could be handy for archiving the generated documentation or if some version
 # control system is used.
 
-PROJECT_NUMBER         = 4.2.0
+PROJECT_NUMBER         = 4.3.1
```

Comment:
Updating this is getting annoying, but not so annoying to actually sit down and figure out how to make it dynamic.

Reply:
https://stackoverflow.com/questions/39581739/doxygen-outputting-version-numbers ?

Reply:
My brain can't process this now - next time maybe I'll look into it